miso-web/src/main/webapp/pages/editLibrary.jsp

```diff
@@ -374,6 +374,15 @@
 <br/>
 <h2>Details</h2>
 <table class="in">
+  <tr>
+    <td class="h">Upstream Group ID:</td>
+    <td id="effectiveGroupId">
+      <c:choose>
+      <c:when test="${empty effectiveGroupId}">(None)</c:when>
+      <c:otherwise>${effectiveGroupId}</c:otherwise>
```

Comment:
Didn't you also want to show the alias of the entity that this is set on? Same on Edit Sample page and the bulk pages

Reply:
Yes! But only on the Edit pages; the bulk pages already have enough columns that I'm going to leave the alias off for now, and we can add it later if it's desired.

Reply:
:+1: